projects/packages/analyzer/src/api/class-analyze-controller.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Automattic\Jetpack\Analyzer;
+
+class Analyze_Controller extends Controller {
+	public function get() {
+		try {
+			Locker::lock();
+		} catch ( \Throwable $th ) {
+			// TODO: extend default Exception class, and use instance_of instead.
+			if ( 'Locked already' === $th->getMessage() ) {
+				return $this->model->get_status();
+			}
+			throw $th;
+		}
+
+		$this->model->reset();
+		$this->model->toggle_status();
+
+		$this->start_scan();
+
+		return $this->model->get_status();
+	}
+
+	private function start_scan() {
+		// TODO: maybe sanitize params?
```

Comment:
If nothing else, at least wrap each one in `escapeshellarg()` where they're being interpolated on line 27.

Or even better, as above if we're requiring PHP 8 now we could make `$cmd` an array and save a lot of escaping.